README.md

```
@@ -48,10 +48,97 @@ export function ExampleScreen() {
+  show('error-modal', 100)
+}
+
+// Hide a specific modal
+function closeModal(modalId: string) {
+  const { hide } = useModalStore.getState()
+  hide(modalId)
+}
+```
+
+**Conditional modal chains:**
+```tsx
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)